packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -166,10 +166,16 @@ export class SummarizerNode implements IRootSummarizerNode {
         // If there is no latestSummary, clearSummary and return before reaching this code.
         assert(!!localPathsToUse, 0x1a5 /* "Tracked summary local paths not set" */);
 
+        // DataStore can be realized during the summary execution (ex. the mixinSummaryHandler
+        // in order to provide search capability to the summaries). If that happens,
+        // a child node will not have the handle paths with ".channels".
+        // By using the _latestSummary's basePath we have a safe path to compose its pendingSummary.
+        // Bug: https://dev.azure.com/fluidframework/internal/_workitems/edit/1633
```

Comment:
We should remove references to ADO since its MS internal. Just keep the link to the PR.